cosmopolitan_continents/migrations/0001_initial.py

```diff
@@ -11,9 +11,9 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
-            name='Continent',
+            name='CosmopolitanContinent',
             fields=[
-                ('id', models.CharField(serialize=False, primary_key=True, max_length=2)),
+                ('id', models.CharField(max_length=2, serialize=False, primary_key=True)),
```

Comment:
from django_cities we have 3 letters code already: in `code3` field (Char, as django_cities offers only one code for one country). Is that enough?